app/controllers/admin_censor_rule_controller.rb

```diff
@@ -63,18 +63,25 @@ def set_info_request_and_censor_rule_and_form_url
       @censor_rule = @censor_user.censor_rules.build(censor_rule_params)
       @form_url = admin_user_censor_rules_path(@censor_user)
     end
+
```

Comment:
Its verbose, but it's doing what it says on the tin. A way to avoid this would be to make `CensorRule` a [polymorphic association](https://github.com/mysociety/alaveteli/issues/2011).